.github/workflows/CI.yml

```diff
@@ -69,13 +68,53 @@ jobs:
         uses: actions/setup-java@v1
         with:
           java-version: 1.8
-      - name: Run UATs
+      - name: Install CLI and dependencies
         run: |
-          coverage run --source=gdk -m pytest -v -s uat
+          pip install .
+          gdk --help
+          pip install -r test-requirements.txt
       - name: Run UATs with code coverage
         run: |
           coverage run --source=gdk -m pytest -v -s uat --instrumented && coverage xml --fail-under=77
       - name: Upload UAT coverage to Codecov
         uses: codecov/codecov-action@v2
         with:
-          flags: uat
\ No newline at end of file
+          flags: uat
+  regression:
```

Comment:
Should we move this into a separate workflo?

Reply:
I am going to revisit this anyways with GDK version regression story. I think we are good with this state in interim.